static/js/search.js

```diff
@@ -57,16 +57,19 @@ limitations under the License.
  }
  
 function getColumns() {
+  //Changed the "availColNames.lenght == 0" to "availColNames.length != 0" as the initial condition did not let the code 
   if (availColNames.length == 0) {
     data = {
       state: "query",
-      searchText: "*",
-      startEpoch: "now-24h",
+      searchText: availColNames.searchText,
+      startEpoch: availColNames.startDate, 
+      //Made changes to this line to fix BUG #250. Initially, the `startEpoch` was assigned static value of "now-24h" due 
```

Comment:
Did  you test this ? This is wrong, you cannot not get search text and start date from available columns.